READMEデータベース設計

suzukinan opened the pull request:
# WHAT
READMEの
テーブル4種類を追加

# WHY
データベース設計のため

suzukinan committed:
readme

READMEの
テーブルを追加

TOMARUMARU commented on README.md:
```diff
@@ -22,3 +22,56 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|Name|string|index: true, null: fales, unique: true|
```
nameで大丈夫です

TOMARUMARU commented on README.md:
```diff
@@ -22,3 +22,56 @@ Things you may want to cover:
+|Email|string|null: false|
+
+### Association
+- has_many :groups, through: ,members
```
細かいですが through: :members です

TOMARUMARU commented on README.md:
```diff
@@ -22,3 +22,56 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|Name|string|null: fales, foreign_key: true|
```
nameで大丈夫です
外部キー制約は不要ですね

TOMARUMARU commented on README.md:
```diff
@@ -22,3 +22,56 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|user_id|integer|null: fales, foreign_key: true|
```
falseのtypoがいくつかあります

suzukinan committed:
タイプミス修正

fales → false
Name → name
through: ,members → through: :members

57行目の外部キー制約の削除いたしました。

suzukinan commented:
ご指摘ありがとうございます。
修正いたしました。

TOMARUMARU reviewed (approved):
LGTM
